site-service/src/test/java/uk/ac/ox/ndph/mts/site_service/repository/HapiFhirRepositoryTests.java

```diff
@@ -53,6 +54,24 @@ void TestHapiRepository_WhenSaveOrganization_SendsBundleWithTransactionType()
         assertThat(type, equalTo(Bundle.BundleType.TRANSACTION));
     }
 
+    @Test
+    void TestHapiRepository_WhenSearchOrganizationByName_Sends_NOT_FOUND()
```

Comment:
The name of the test should be adjusted to reflect the change in returned value.

Reply:
Addressed as discussed